OJS.Workers.ExecutionStrategies/Sql/MySql/BaseMySqlExecutionStrategy.cs

```diff
@@ -63,8 +64,19 @@ public override IDbConnection GetOpenConnection(string databaseName)
                 this.ExecuteNonQuery(connection, grandPrivilegesToUserQuery);
             }
 
-            var createdDbConnectionString =
-                $"Server=localhost;Database={databaseName};UID={this.restrictedUserId};Password={this.restrictedUserPassword};Pooling=False;";
```

Comment:
If we want such a scenario, we would have to redesign the whole strategy, since the implementation requires only False... it can not support the other. Every user uses it's own unique database, and pooling is not going to be beneficial.
Configuration value is a good option if it is possible to change just that to alter some behavior, but everything else in the code should stay the same. But nevertheless good way of thinking!